### DISCUSSION TITLE
Add recipe for storage/tank unit + storage upgrader to upgrade the items on a workbench

### DISCUSSION THREAD

**Discussion by rapus95** [TechReborn/TechReborn#3001] [Ideas]
that way I don't have to place them down

**Comment by Ayutac** [comment]
This is a bad idea. Workbench recipes cannot work with NBT, so the result would always be an empty unit. There is no proper way to warn people about that / people are gonna make the mistake anyway.

If you want to upgrade empty units, just use the old recipes. If you want to upgrade a unit standing somewhere with content, use the upgrader. Placing down the unit where you want it to be anyway isn't a hassle.
